step-capstone/src/styles/TimeLine.css

```diff
@@ -1,3 +1,21 @@
+  
+* {
+  margin: 0;
+  border: 0;
+}
+body {
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  font-family: "DINPro", "Helvetica Neue", sans-serif;
+  margin: 0;
+  background: #fafafa;
+  box-sizing: border-box;
+  height: 100vh;
+
+}
```

Comment:
this shouldn't be added back right?

Reply:
woop, that should be deleted. Thank you Zach!

Reply:
addressed